src/components/Footer/__tests__/Footer.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import { render, fireEvent } from "@testing-library/vue";
+import Footer from "./../";
+
+describe("components/Footer", () => {
+  it("calls handleClick callback on click", async () => {
+    const spy = jest.fn();
+    const copyright = "this is my copyright";
+    const { getByTestId } = render(Footer, {
+      props: {
+        copyright: copyright,
+        links: [
+          {
+            referenceId: 1,
+            referenceType: "page",
+            isActive: true,
+            label: "footer1",
+          },
+          {
+            referenceId: 2,
+            referenceType: "page",
+            isActive: false,
+            label: "footer2",
+          },
+        ],
+        handleClick: spy,
+      },
+    });
+    const footer = getByTestId("footer-1");
+    await fireEvent(footer!, new Event("click"));
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it("renders content which is passed in", () => {
+    const spy = jest.fn();
+    const copyright = "this is my copyright";
+    const { getByTestId } = render(Footer, {
+      props: {
+        copyright: copyright,
+        links: [
+          {
+            referenceId: 1,
+            referenceType: "page",
+            isActive: true,
+            label: "footer1",
+          },
+          {
+            referenceId: 2,
+            referenceType: "page",
+            isActive: false,
+            label: "footer2",
+          },
+        ],
+        handleClick: spy,
+      },
+    });
+    const copyRightTest = getByTestId("footer-copyright");
+    expect(copyRightTest).toBeTruthy();
```

Comment:
Please change test to check for correct text and add tests for the footer-links / logo as well.